packages/itwinui-css/backstop/tests/input.html

```diff
@@ -502,5 +505,118 @@ <h2>No wrapping label</h2>
       </div>
 
     </section>
+
+    <br />
+
+    <h2>Search box</h2>
```

Comment:
In the react side, you have examples with `startAdornment`. You could maybe have some examples of `startAdornment` in the css html page too.